docs/doxygen/pages.dox

```diff
@@ -21,10 +21,10 @@ This PKCS #11 library implements a subset of the PKCS #11 API required to establ
 
 /**
 @page pkcs11_design Design
-@section PKCS11_Wrapper PKCS #11 Wrapper Dependencies
-Currently, the PKCS #11 wrapper library has a dependency on:
-- The C standard library stdint
-- PKCS #11.
+@section PKCS11_Wrapper Dependencies of corePKCS11 Wrapper for PKCS #11 standard
+Currently, the corePKCS11 library provides a wrapper header around the PKCS #11 standard. It has depdnencies on:
+- The C standard library `stdint`
+- PKCS #11 standard.
```

Comment:
Perhaps this could be a link to the spec from OASIS to make it more clear?

Reply:
Added link to OASIS